pkg/endpoint/bpf.go

```diff
@@ -765,6 +766,11 @@ func (e *Endpoint) regenerateBPF(owner Owner, currentDir, nextDir string, regenC
 		return 0, compilationExecuted, fmt.Errorf("unable to regenerate policy because PolicyMap synchronization failed: %s", err)
 	}
 
+	err = eppolicymap.WriteEndpoint(epInfoCache.keys, e.PolicyMap.Fd)
+	if err != nil {
+		e.logStatusLocked(BPF, Warning, fmt.Sprintf("Unable to sync EpToPolicy Map continue with Sockmap support: %s", err))
```

Comment:
Why not return here?

Reply:
So if the endpoint to policymap is not populated everything continues to work as before we just don't use sockmap on this endpoint.

Reply:
What are the side effects of not using sockmap? Will we continue to enforce L7 policy OK, just without the sockmap lookup?

Reply:
Correct, the current default policy for sockmap, if it can not find a policy or the destination is not an endpoint on the node, is to pass the message to the TCP stack. This will then go through the existing L7/L4/L3 policy implemented at the lower layers. We will lose the performance benefits but it is functionally correct. When we add debug/monitoring in a followup PR we probably want to track these so we can detect these failures for dbg reasons but functionally everything should work as expected.